### PR TITLE
add legal and impressum

### PR DESCRIPTION
**Pull request by Carolin-Weitkamp**
still not styled.... this will follow later

### CODE REVIEW

File: pages/impressum.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  );
+}
+
+const Hosts = styled.div`
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  grid-template-columns: repeat(6, [col-start] 1fr);
+  grid-template-rows: repeat(14, [col-start]);
+`;
```
Comment (PReichetanz) [inline pages/impressum.js:53]: I don't understand why you need this if you only have one child.

File: pages/privacy.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+export default function privacy() {
```
Comment (PReichetanz) [inline pages/privacy.js:3]:
```suggestion
export default function Privacy() {
```

File: pages/privacy.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <Hosts>
+      <p>Datenschutz</p>
```
Comment (PReichetanz) [inline pages/privacy.js:6]: This is a heading and thus a heading HTML element should be used.

File: pages/privacy.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  );
+}
+
+const Hosts = styled.div`
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  grid-template-columns: repeat(6, [col-start] 1fr);
+  grid-template-rows: repeat(14, [col-start]);
+`;
```
Comment (PReichetanz) [inline pages/privacy.js:37]: Same as in impressum.js.

**Review by PReichetanz — COMMENTED** [review]
I left some comments to consider.

File: pages/impressum.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+export default function impressum() {
```
[the remark this answers is not in the document]
Comment (Carolin-Weitkamp) [reply pages/impressum.js:3]: merci!

File: pages/impressum.js (hunk 1 above)
Comment (Carolin-Weitkamp) [reply pages/impressum.js:53]: Can you explain it diffrently, cause i dont get it :) 
I want to use it for the styling, via Styled Components.

Comment (Carolin-Weitkamp) [reply pages/impressum.js:53]: for now i replaced it with "div"....

File: pages/privacy.js (hunk 4 above)
Comment (Carolin-Weitkamp) [reply pages/privacy.js:37]: Thanks a lot for your recommendations!

File: pages/impressum.js (hunk 1 above)
Comment (PReichetanz) [reply pages/impressum.js]: I don't mind the div, but why do you need a grid with several columns and rows if you only have one child element inside this element

**Comment by Carolin-Weitkamp** [comment]
He Paul, i got it and i deleted the Style completly now :)

Thanks!

**Pull request merged**